guide-us-english.tex

```diff
@@ -263,8 +263,6 @@ \subsection{Scaling the Product Owner Team}
 
 \includegraphics[width=1.0\linewidth]{MetaScrum-R2.png}
 
-\textbf{NOTE:} As mentioned above, these pentagons represent the ideal sized Scrum Teams and ideal sized Product Owner Teams. These diagrams are meant to be examples only, your organizational diagram may differ greatly.
```

Comment:
On that note, is there mentioned anywhere that the chosen default is 5? Maybe this warrants a separate paragraph somewhere so we do not need to note it everywhere else. Just put that previous note in a paragraph "Note on Team size"